utils/githooks/pre-commit.d/10-update-copyright

```diff
@@ -2,6 +2,12 @@
 
 # A git hook to validate and correct the copyright date in source files.
 
+if [ -e .git/MERGE_HEAD ]
```

Comment:
It might be useful to have a way to bypass this so we can update copyrights on files that have changed without the githooks.

RUN_GITHOOK_ON_MERGE=1 git commit -a - s or some such